### ISSUE TITLE
Missing support for direct rollup configuration

### ISSUE THREAD

**Issue by tivac** [nolanlawson/rollupify#7]
It would be nice if I could directly configure rollup from code instead of having to specify my rollup config in a `rollup.config.js` file.

Right now I have:

``` js
builder.transform("rollupify", { config : "./rollup.config.js" });
```

but I'd really prefer

``` js
builder.transform("rollupify", {
    config : {
        plugins : [ ... ]
        onwarn : function() { ... }
    }
});
```

Is that something you'd be willing to accept a PR for? I almost always write an actual JS file to drive browserify and would prefer to keep the configuration in that same code instead of scattered around.

**Comment by nolanlawson** [comment]
Yep, that sounds reasonable. It seems like we could support this by simply checking whether the value of `config` is a string or an object. Happy to accept a PR here. :)

**Comment by tivac** [comment]
Closed by #8 and `rollupify@0.3.0`!

:+1:
